### PR TITLE
Xode_Suggestions_2022_01_05

### PR DESCRIPTION
**Pull request by Xode672**

### CODE REVIEW

File: script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_2_2.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <Page1>
 ...I drop my gaze to the floor as if to turn away from the ominous premonition. //Avoiding repetition
 <Page2>
-...This is fine.
+...This is fine
```
Comment (Xode672) [inline script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_2_2.txt:4]: Ignore this I don't have a clue on how to remove this test change from my pull requests.

Comment (Avuxo) [reply script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_2_2.txt:4]: Just go back into the file and add the period back. You can then recommit with the change.
:arcsmile:

File: script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_6.txt [hunk 2]
```diff
@@ -21,7 +21,7 @@ Awooooooooooo.
 Awooooooooooo――――
 <Page7>
 "...Ah, enough, just shut up already!"
-I open my eyes, putting on my glasses.
+I open my eyes, and put on my glasses.
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_6.txt:24]: drop comma

File: script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_6.txt [hunk 3]
```diff
@@ -144,8 +144,8 @@ On the way, I see something I don't like.
 Even though I'm wearing my glasses, I can still see those 'lines'.
 <Page34>
 "Ugh......!"
-My eyeballs can't take the nausea that's been building up.
-It's a reaction to the fact that I've been trying not to see them for the last seven years.
+My eyeballs can't take the nausea that's building up.
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_6.txt:147]: revert imo

File: script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_6.txt [hunk 4]
```diff
@@ -144,8 +144,8 @@ On the way, I see something I don't like.
...
+It's the consequence/reprecussion to the fact that I've been trying not to see them for the last seven years.
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_6.txt:148]: Not a big fan of this change either. Ignoring the 'consequence to' syntax, I think reaction is more correct.

**Review by Avuxo — CHANGES_REQUESTED** [review]

**Comment by rschlaikjer** [comment]
Updated to new script format
https://github.com/Tsukihimates/Tsukihime-Translation/pull/250

**Pull request closed without merging**